onboarding.md

```diff
@@ -207,8 +207,7 @@ needs to be pointed out separately during the onboarding.
 * Optionally, include your personal pronouns.
 * Label your pull request with the `doc`, `notable-change`, and `fast-track`
   labels.
-* Run CI on the PR. Because the PR does not affect any code, use the
-  `node-test-pull-request-lite-pipeline` CI task.
+* Run CI on the PR. Use the `node-test-pull-request-pipeline` CI task.
```

Comment:
There's no such job. You probably mean `node-test-pull-request`? Unfortunately that will take a lot longer to complete than the obsoleted lite jobs.

Reply:
Fwiw, that’s what I always use during onboardings.

Reply:
@richardlau I always have people start that job (so they know how to start a CI job) and then cancel it (because, yeah, not needed for the onboarding PR).

Reply:
@Trott Where possible I'd discourage cancelling jobs as it can leave git workspaces in bad states.

Reply:
@richardlau That's indeed unfortunate. Oh well.